conf.d/unbound.yaml.example

```diff
@@ -0,0 +1,5 @@
+init_config:
+  # Not required for this check
```

Comment:
👍 added that and a comment. I wasn't sure whether a URL in the comment was appropriate, but if you go to the "SETUP" portion of the man page for `unbound-control`, it describes the cert generation of the certs: https://www.unbound.net/documentation/unbound-control.html